monai/transforms/meta_utility/dictionary.py

```diff
@@ -78,13 +79,24 @@ class ToMetaTensord(MapTransform, InvertibleTransform):
 
     backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
 
+    def __init__(self, keys: KeysCollection, meta_keys: Optional[KeysCollection] = None) -> None:
```

Comment:
added a `meta_keys` arg, so that essentially we can have `Metatensor(d[key], meta=d['meta_key'])` (it's useful for SaveImaged), cc @rijobro